velox/functions/remote/client/CMakeLists.txt

```
@@ -16,11 +16,23 @@ velox_add_library(velox_functions_remote_thrift_client ThriftClient.cpp)
 velox_link_libraries(velox_functions_remote_thrift_client
                      PUBLIC remote_function_thrift FBThrift::thriftcpp2)
 
+set(curl_SOURCE BUNDLED)
+velox_resolve_dependency(curl)
+
+velox_add_library(velox_functions_remote_rest_client RestClient.cpp)
+velox_link_libraries(velox_functions_remote_rest_client Folly::folly
+                     ${CURL_LIBRARIES})
```

```suggestion
                    CURL::libcurl)
```
Always prefer targets over variables.